packages/libraries/client/package.json

```diff
@@ -54,16 +54,16 @@
   },
   "optionalDependencies": {
     "@apollo/server": "^4.0.0",
-    "@envelop/types": "^4.0.0"
+    "@envelop/types": "^5.0.0"
   },
   "devDependencies": {
     "@apollo/federation": "0.38.1",
     "@apollo/server": "4.9.5",
     "@apollo/subgraph": "2.5.6",
-    "@envelop/types": "4.0.1",
+    "@envelop/types": "5.0.0",
     "@types/async-retry": "1.4.7",
     "graphql": "16.8.1",
-    "graphql-yoga": "4.0.4",
+    "graphql-yoga": "5.0.0",
     "nock": "13.3.6",
     "vitest": "0.34.6"
   },
```

Comment:
If we bump this here, we also need to release a new version, can you add a changeset @EmrysMyrddin?

Reply:
Done!